js/app.js

```diff
@@ -1,22 +1,31 @@
-var hobbies = [];
 
-function Hobby (opts) {
+var projects = [];
+
+function Project (opts) {
   this.title = opts.title;
   this.body = opts.body;
 }
 
-Hobby.prototype.toHtml = function() {
-  var $newHobby = $('article.template').clone();
-  $newHobby.find('h2').html(this.title);
-  $newHobby.find('.hobbies-body').html(this.body);
-  $newHobby.removeClass('template');
-  return $newHobby;
+Project.prototype.toHtml = function() {
+  // var $newProject = $('article.template').clone();
+  // $newProject.removeClass('template');
+  // $newProject.find('h2').html(this.title);
+  // $newProject.find('.projects-body').html(this.body);
+  // $newProject.removeClass('template');
+  // return $newProject;
```

Comment:
you can take this commented out code now if it's working so it doesn't busy up your file too much :)